rolling/distribution.yaml

```diff
@@ -1943,6 +1943,21 @@ repositories:
       url: https://github.com/KIT-MRT/mrt_cmake_modules.git
       version: master
     status: maintained
+  nao_button_sim:
```

Comment:
this package was being released in previous distributions from another repo https://github.com/ros/rosdistro/blob/c833e0cb5ca5d2507bb83f7962cf44cd23dee4b6/galactic/distribution.yaml#L1996?
is the change intentional?

Reply:
This comment slipped past, yes, this was intentional.